pepper-apis/src/main/java/org/broadinstitute/ddp/event/HousekeepingTaskReceiver.java

```diff
@@ -85,6 +89,13 @@ private void handleClearCache(PubsubMessage message, AckReplyConsumer reply) {
         reply.ack();
     }
 
+    private void handleClearRedis(PubsubMessage message, AckReplyConsumer reply) {
+        LOG.info("Clearing redis caches used in Housekeeping...");
+        CacheService.getInstance().resetAllCaches();
```

Comment:
I think it will be nice to have this feature to "clear" Redis caches. HOWEVER, note that this does NOT clear ALL Redis caches. If you look through the code, you'll see that this `resetAllCaches()` method will only clear caches that has been _initialized_. So calling this method here will only clear caches that has been used by Housekeeping. Hopefully the log message of `... used in Housekeeping...` makes this explicit.

Also note that this `clear_redis` message is different from the `clear_cache` message. The former is clearing stuff in Redis, while the latter is clearing stuff stored _in memory_.

Here's a screenshot of the caches that this will clear in Housekeeping after a Brain Study export (there might be more if you do `clear_redis` after Housekeeping had used other caching code):

![Screen Shot 2021-01-04 at 7 34 47 PM](https://user-images.githubusercontent.com/5713833/103594144-cf65a100-4ec5-11eb-91c0-14a907d91c1c.png)